lib/src/simple_op_store.rs

```diff
@@ -69,15 +71,18 @@ impl SimpleOpStore {
     pub fn init(store_path: &Path) -> Self {
         fs::create_dir(store_path.join("views")).unwrap();
         fs::create_dir(store_path.join("operations")).unwrap();
-        SimpleOpStore {
-            path: store_path.to_owned(),
-        }
+        let store = Self::load(store_path);
+        let empty_view_id = store.write_view(&View::default()).unwrap();
+        assert_eq!(store.empty_view_id, empty_view_id);
```

Comment:
Needs a virtual hash?

Suppose we add a new field to the View object, the `empty_view_id` will have to be updated. Then, the existing op store won't have the new empty view object to read.